### PR TITLE
Requirement Enhancements

### PR DESCRIPTION
**Pull request by lxatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
- Upgrade to ReqIfSharp version 4.0.0
- Inform the user about Model Validation failures during Reqif Export

### CODE REVIEW

**Review by alexatrhea — APPROVED** [review]

File: Requirements.Tests/ReqIF/ReqIFThingFactoryTestFixture.cs [hunk 1]
```diff
@@ -61,9 +87,11 @@ internal class ReqIFThingFactoryTestFixture
         private BinaryRelationshipRule reqRule;
 
         private ParameterizedCategoryRule parameterRule;
+
         #endregion
```
Comment (samatrhea) [inline Requirements.Tests/ReqIF/ReqIFThingFactoryTestFixture.cs:91]: please remove all regions

**Review by samatrhea — APPROVED** [review]
before landing, can you please remove the regions? for the rest the code looks good to me

File: Requirements.Tests/ReqIF/ReqIFThingFactoryTestFixture.cs (hunk 1 above)
Comment (lxatrhea) [reply Requirements.Tests/ReqIF/ReqIFThingFactoryTestFixture.cs:91]: Will do in PR #993

**Comment by MartinFijneman** [comment]
verified in testing with 9.1.0 RC2

**Pull request merged**